src/processor/processor.module.ts

```diff
@@ -25,27 +25,11 @@ import { TxMonitorQueueConsumerService } from './tx-monitor-queue-consumer.servi
   imports: [
     EventEmitterModule,
     BullModule.forRootAsync({
-      imports: [ConfigModule],
-      useFactory: (configService: ConfigService) => {
-        // Note: BullMQ doesn't honor a URL for the Redis connection, and
-        // JS URL doesn't parse 'redis://' as a valid protocol, so we fool
-        // it by changing the URL to use 'http://' in order to parse out
-        // the host, port, username, password, etc.
-        // We could pass REDIS_HOST, REDIS_PORT, etc, in the environment, but
-        // trying to keep the # of environment variables from proliferating
-        const url = new URL(configService.redisUrl.toString().replace(/^redis[s]*/, 'http'));
-        const { hostname, port, username, password, pathname } = url;
-        return {
-          connection: {
-            host: hostname || undefined,
-            port: port ? Number(port) : undefined,
-            username: username || undefined,
-            password: password || undefined,
-            db: pathname?.length > 1 ? Number(pathname.slice(1)) : undefined,
-          },
-        };
-      },
-      inject: [ConfigService],
+      useFactory: (redis: ReconnectionCacheMgrService) => ({
+        enableReadyCheck: true,
+        connection: redis.redis,
```

Comment:
Switch to use the redis from the `ReconnectionCacheMgrService`. I thought about making redis a separate service, but I ended up doing this for now.